app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -4,8 +4,8 @@ module ManageIQ::Providers::Kubernetes::ContainerManagerMixin
   extend ActiveSupport::Concern
 
   module ClassMethods
-    def raw_api_endpoint(hostname, port)
-      URI::HTTPS.build(:host => hostname, :port => port.presence.try(:to_i))
+    def raw_api_endpoint(hostname, port, path = '')
```

Comment:
shouldn't we change also the call [down here](https://github.com/ManageIQ/manageiq/pull/11057/files#diff-dc5b8d4fccab9961c5a3ae22c89e8ca1R20) to extract the path from the ```options``` argument and pass it here?